app/controllers/users_controller.rb

```diff
@@ -4,5 +4,20 @@ def show
     @user = User.find_by!(username: params.fetch(:username))
   end
 
-end
+  def liked
+    @user = User.find_by!(username: params.fetch(:username))
+  end
+
+  def feed
```

Comment:
User Controller looks great. For consistency I would recommend using either `[]` or `()` in this case.